src/core/Form/Checkbox/CheckboxGroup.tsx

```diff
@@ -76,9 +102,24 @@ class BaseCheckboxGroup extends Component<
             {groupHintText && <HintText>{groupHintText}</HintText>}
           </HtmlLegend>
           <HtmlDiv className={checkboxGroupClassNames.container}>
-            {children}
+            <Provider
+              value={{
+                status: groupStatus,
+              }}
+            >
+              {children}
+            </Provider>
           </HtmlDiv>
         </HtmlFieldSet>
+        <StatusText
+          className={classnames({
+            [checkboxGroupClassNames.statusTextHasContent]: !!groupStatusText,
+          })}
+          id={statusTextId}
+          status={groupStatus}
+        >
+          {groupStatusText}
+        </StatusText>
```

Comment:
This more a comment on the visual design: The group status text could be a bit difficult to connect to the right context when it's just under the last checkbox. At least I could confuse it to only be connected to the last box, not the entire group

Reply:
As discussed elsewhere, will leave the decision to be made by the user of the library.

In normal cases when using Checkboxes in a group, the `groupStatusText` will be the only one used.